Resources/Prototypes/Roles/Jobs/Engineering/atmospheric_technician.yml

```diff
@@ -4,9 +4,9 @@
   description: job-description-atmostech
   playTimeTracker: JobAtmosphericTechnician
   requirements:
-    - !type:DepartmentTimeRequirement
-      department: Engineering
-      time: 54000 # 15 hrs
+  - !type:RoleTimeRequirement
+    role: JobStationEngineer
+    time: 9000 #2.5 hrs
```

Comment:
imo should be unlocked the same time as engineering. You should be familiar with the basics as a TA for building and power and pipes.